LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-at-rule/empty-string-symbol.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>CSS Test: counter-style empty string symbol</title>
+<link rel="help" href="https://drafts.csswg.org/css-counter-styles-3/#typedef-symbol">
+<link rel="mismatch" href="empty-string-symbol-ref.html">
```

Comment:
For mismatch files, WPT naming convention is `-notref.html`, can you rename the `-ref.html` file and update this reference?